website/docs/guides/markdown-features/markdown-features-admonitions.mdx

```diff
@@ -69,6 +69,27 @@ Some **content** with _markdown_ `syntax`. Check [this `api`](#).
 
 :::
 
+## Usage with Prettier
+
+If you use [Prettier](https://prettier.io) to format your Markdown files, Prettier might autoformatting your code which results in invalid the admonition syntax. To avoid this problem, add empty lines around your content. This is also why the examples we show here all have empty lines around the content.
```

Comment:
```suggestion
If you use [Prettier](https://prettier.io) to format your Markdown files, Prettier might autoformat your code to invalid admonition syntax. To avoid this problem, add empty lines around the starting and ending directives. This is also why the examples we show here all have empty lines around the content.
```

Reply:
Thank you, I should have proofread it 🤦🏻